packages/fluent-theme/src/fluent/styles/DefaultButton.styles.ts

```diff
@@ -68,7 +68,8 @@ export const DefaultButtonStyles = (props: IButtonProps): Partial<IButtonStyles>
           }
         },
         '.ms-Button.is-disabled': {
-          backgroundColor: palette.neutralLighter
+          backgroundColor: palette.neutralLighter,
+          color: palette.neutralTertiary
```

Comment:
Is this supposed to be the missing label? I thought by "adding missing label", it'd mean something else and not a color.

Reply:
I do see that "create account" appears though I don't understanding how setting this color makes that happen...